### PR TITLE
[NONMODULAR] Clown slots can no longer be edited.

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
Title

## Why It's Good For The Game
Host request for policy reasons

## Changelog
:cl:
fix: Clown slots can no longer be edited above roundstart limit (0)
/:cl:

### CODE REVIEW

**Comment by Vizoplez** [comment]
dont merge yet I also need to check if this applies for the tablet computer

**Comment by GoldenAlpharex** [comment]
While you're at it, can you look into the console being able to edit the slots that are above the max that's in the config, to allow the HoP to deal with the Bureaucratic Errors?

**Comment by Vizoplez** [comment]
> While you're at it, can you look into the console being able to edit the slots that are above the max that's in the config, to allow the HoP to deal with the Bureaucratic Errors?

unfortunately this is a code issue I cant really touch, as I would need to rewrite how the blacklist works, and /tg/ keep changing it :AAA:

**Comment by Vizoplez** [comment]
teaches me to phone edit. Moment

**Comment by tf-4** [comment]
Is there a reason you aren't adding prisoner to the modular version, or just an oversight?

**Comment by Vizoplez** [comment]
> Is there a reason you aren't adding prisoner to the modular version, or just an oversight?

prisoner slots was a holdover from oldbase, and as it stands a few maps have inconsistent amounts of prisoner slots which may need to be raised/lowered based on alert.

EDIT: Might add it back though. Depends on what the host/maintainers want with prisoners, they're weird in terms of job priority.

File: code/modules/modular_computers/file_system/programs/jobmanagement.dm
Comment (KathrinBailey) [inline code/modules/modular_computers/file_system/programs/jobmanagement.dm]: Huh?

Comment (Vizoplez) [reply code/modules/modular_computers/file_system/programs/jobmanagement.dm]: oop

**Review by KathrinBailey — APPROVED** [review]

**Pull request merged**
